source/isaaclab/isaaclab/sim/spawners/wrappers/wrappers.py

```diff
@@ -5,14 +5,15 @@
 
 from __future__ import annotations
 
-import random
 import re
 from typing import TYPE_CHECKING
 
 import carb
 import isaacsim.core.utils.prims as prim_utils
 import isaacsim.core.utils.stage as stage_utils
 from isaacsim.core.utils.stage import get_current_stage
+
+from hydra.utils import get_method
```

Comment:
Would avoid using hydra inside the core base. Hydra is an optional dependency used only for env/agent configuration.